core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -717,6 +718,17 @@ public static String getSessionServiceUrl()
     {
         return sessionServiceURL;
     }
+    
+    public static String getOncoKBPublicApiUrl()
+    {
+        String oncokbApiUrl = properties.getProperty(ONCOKB_PUBLIC_API_URL);
+        if(oncokbApiUrl == null || oncokbApiUrl.isEmpty()) {
+            return "";
```

Comment:
And if show.oncokb properry is set to false, then we should return empty string.